net/README.md

```diff
@@ -3,20 +3,24 @@
 ### Pre-requisites
 1. You need Ubuntu 15 or Centos 7 on each of your servers used for the Contiv cluster
 
-2. Do "export CLUSTER_NODE_IPS=ips of all nodes in cluster", and "export no_proxy=ips of all nodes in cluster,127.0.0.1,localhost,netmaster" in your shell
+2. Make sure SELinux is disabled on your nodes in cluster. Use 'set enforce 0' command to do this. For more information please take a look [here](https://www.centos.org/docs/5/html/5.2/Deployment_Guide/sec-sel-enable-disable-enforcement.html)
 
-3. If your servers are behind an http proxy (usually the case in many cisco labs...), you need
+3. Make sure you are running ansible 2.0.x version on these nodes.
```

Comment:
Gaurav, I think the installer should take care of this as it already tries to install ansible if it is not installed.